web/src/app/alerts/CreateAlertDialog/useCreateAlerts.ts

```diff
@@ -36,21 +36,23 @@ const getAliasedMutation = (
 export const useCreateAlerts = (
   value: Value,
 ): [MutationFunction, MutationResult, (alias: string | number) => string] => {
+  const sids = value?.serviceIDs ?? []
+
   // 1. build mutation
   let m = getAliasedMutation(baseMutation, 0)
-  for (let i = 1; i < value.serviceIDs.length; i++) {
+  for (let i = 1; i < sids.length ?? 0; i++) {
```

Comment:
`sids` is a const and already defaults to `[]`

```suggestion
  for (let i = 1; i < sids.length; i++) {
```

Reply:
I was seeing failures at one point, but I'll revert this and see how it behaves